experimental/shared/src/main/scala/zio/prelude/experimental/BothCompose.scala

```diff
@@ -57,6 +59,22 @@ object BothCompose {
     }
 
   }
+
+  implicit val URIOApplicationCompose: BothCompose[URIO] = new BothCompose[URIO] {
```

Comment:
Shouldn't we just define this instance in `AssociativeCompose` based on our usual organization strategy?

Reply:
We can't currently do that, because `AssociativeCompose` is in _core_, while `BothCompose` is in _experimental_.

Or we could move `BothCompose` to _core_ and do it there, I'm all for it :wink:

Reply:
Ah I see.